### ISSUE TITLE
[1.2.0] SyntaxError: Unexpected token )

### ISSUE THREAD

**Issue by dkokic** [nestjs/swagger#68]
The following is the error log after running the sample code:

```
/tmp/product-catalog-ts/node_modules/@nestjs/swagger/decorators/api-operation.decorator.js:21
    ),
    ^

SyntaxError: Unexpected token )
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Module._extensions..js (module.js:579:10)
    at Object.require.extensions.(anonymous function) [as .js] (/tmp/product-catalog-ts/server/node_modules/ts-node/src/index.ts:392:14)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)

```

The problem seems to be caused by trailing commas and is not present in the release 1.1.4

Is this an issue with my TypeScript setup as the trailing commas should be allowed?!

**Comment by kamilmysliwiec** [comment]
Thanks for reporting! Please, install `1.2.1` and lemme know if bug still appears.

**Comment by gureevcr** [comment]
As for me problem still appears on `1.2.1` using `node@6`. But is not presented on `node@8`

**Comment by kamilmysliwiec** [comment]
Got it! Thanks @gureevcr, fixed in `1.2.2`.
